packages/contracts-bedrock/contracts/universal/StandardBridge.sol

```diff
@@ -20,6 +19,11 @@ import { MantleMintableERC20 } from "./MantleMintableERC20.sol";
 abstract contract StandardBridge {
     using SafeERC20 for IERC20;
 
+
+    uint32 constant BIT_TX = 1;
+    uint32 constant ERC20_TX = 2;
+    uint32 constant ETH_TX = 0;
```

Comment:
Can we move this to library? Thus in other contracts hard code types can be avoided.

Reply:
ok,I will finish it

Reply:
fixed on latest commit.